WDAC-Policy-Wizard/app/Resources/policyTemplates/Recommended_Driver_Blocklist.xml

```diff
@@ -2088,14 +4664,9 @@
     </Setting>
     <Setting Provider="PolicyInfo" Key="Information" ValueName="Id">
       <Value>
-        <String>10.0.25210.0</String>
```

Comment:
This needs to remail in the template

Reply:
Line 4667 had this:

<String>10.0.27685.0</String>

Does the version need to match 10.0.25210.0 for something with WDAC Wizard? If so, should <VersionEx> on Line 3 also match the same?

Once I edit this last part for you, I'll se if I can figure out how to update the PR :p